android/src/main/java/cn/mandata/react_native_mpchart/MPPieChartManager.java

```diff
@@ -37,6 +37,12 @@ public void setHoleRadius(PieChart chart, float holeRadius){
         chart.invalidate();
     }
 
+    @ReactProp(name="backgroundColor", defaultInt = Color.WHITE)
+    public void setBackgroundColor(){
```

Comment:
Isn't this already present in MPPieRadarChartManager.java [here](https://github.com/hongyin163/react-native-chart-android/blob/master/android/src/main/java/cn/mandata/react_native_mpchart/MPPieRadarChartManager.java#L64) ?
Just removing the line `chart.setBackgroundColor(Color.WHITE)` like you did below should be enough.